community/modules/scheduler/schedmd-slurm-gcp-v6-controller/cleanup.tf

```diff
@@ -26,10 +41,5 @@ resource "null_resource" "cleanup_compute" {
     when    = destroy
   }
 
-  depends_on = [
-    # Depend on controller network, as a best effort to avoid
-    # subnetwork resourceInUseByAnotherResource error
-    # NOTE: Can not use nodeset subnetworks as "A static list expression is required"
-    var.subnetwork_self_link,
-  ]
+  depends_on = [null_resource.cleanup_compute_depenencies]
```

Comment:
Terraform docs: _We recommend always including a comment that explains why using depends_on is necessary._.
Please add a comment.